### DISCUSSION TITLE
Creating a free tier cluster is unavailable at this time in region westeurope

### DISCUSSION THREAD

**Discussion by MJonker** [Azure-Samples/graphrag-accelerator#72] [Q&A] [answered]
Hi,

I am looking for the place where I can change the tier for the AKS cluster. The deployment script fails with this message:
`\"message\": \"Creating a free tier cluster is unavailable at this time in region westeurope. To create a new cluster, we recommend using an alternate region, or create a paid tier cluster`

I am also surprised that it uses westeurope, in the deployment pararmeters json I have specified swedencentral as the region.

**Comment by MJonker** [comment]
ok, this is free by design:

```To help you get started with AKS, the managed Kubernetes service is free. You only pay for the VMs in your cluster, and any infrastructure resources consumed like storage and networking. This means on Azure, you will pay nothing for the management of your Kubernetes cluster.```

**Comment by timothymeyers** [comment]
@MJonker 

> I am also surprised that it uses westeurope, in the deployment pararmeters json I have specified swedencentral as the region.

Check your resource group region. The deployment will inherit the location from your resource group regardless of what you put in deploy.parameters.json. I think this could be clearer in our Deployment Guide and I may submit an issue or PR related to that.

**Comment by timothymeyers** [comment, marked as answer]
If you'd like to change the tier of the AKS cluster deployment, you'd do it in this [file](https://github.com/Azure-Samples/graphrag-accelerator/blob/main/infra/core/aks/aks.bicep).

Here is the [spec for adding it](https://learn.microsoft.com/en-us/azure/templates/microsoft.containerservice/managedclusters?pivots=deployment-language-bicep#resource-format), look for `sku`. When omitted, it defaults to free I believe.

**Reply by MJonker** [reply]
thanks, I am step further now.
Now cosmos db deployment fails due to  `high demand in West Europe region`
I'll try to change it to north europe #tobecontinued

**Reply by MJonker** [reply]
Thanks again for your help, it is all working now.
I am afraid my Visual Studio Enterprise Subscription won't last a whole month, but there is a lot to learn from this repo!
